datafusion/common/src/dfschema.rs

```diff
@@ -872,42 +890,38 @@ mod tests {
         let left = DFSchema::try_from_qualified_schema("t1", &test_schema_1())?;
         let right = DFSchema::try_from(test_schema_1())?;
         let join = left.join(&right);
-        assert!(join.is_err());
-        assert_eq!(
-            "Schema error: Schema contains qualified \
-        field name \"t1\".\"c0\" and unqualified field name \"c0\" which would be ambiguous",
-            &format!("{}", join.err().unwrap())
-        );
+        assert_contains!(join.unwrap_err().to_string(),
```

Comment:
I switched to `assert_contains` as that includes the expected and actual in the message where `assert!(..contains())` just says "false" when it fails